tensorboard/plugins/hparams/tf_hparams_query_pane/tf-hparams-query-pane.html

```diff
@@ -638,48 +631,31 @@
       _computeSchema() {
         return {
           hparamColumns: this._hparams.map(
-              hparam => ({hparamInfo: hparam.info,
-                          displayed: hparam.displayed})),
+              hparam => ({hparamInfo: hparam.info})),
           metricColumns: this._metrics.map(
-              metric => ({metricInfo: metric.info,
-                          displayed: metric.displayed})),
-          visibleSchema: this._computeVisibleSchema(),
-          loaded: true,
+              metric => ({metricInfo: metric.info})),
         };
       },
 
-      _hparamVisibilityChanged(event) {
-        const hparamIndex = event.model.index;
-        // We'd like the notifications for the changed schema to be sent to
-        // consuming elements after both this.schema.hparamColumns and
-        // schema.visibleSchema are changed so that the schema is in a coherent
-        // state. Thus we first silently modify hparamColumns,
-        // then visibleSchema--which will also send the notification for the
-        // visibileSchema change--and then call notifyPath to send the
-        // notification for the hparamColumns change.
-        this.schema.hparamColumns[hparamIndex].displayed =
-            this._hparams[hparamIndex].displayed;
-        this._updateVisibleSchema();
-        this.notifyPath(`schema.hparamColumns.${hparamIndex}.displayed`);
+      _hparamVisibilityChanged() {
```

Comment:
Instead of this, I would recommend using idiomatic Polymer: http://go/gh/erzel/tensorboard/pull/2/files#diff-c777620714a33699c59f411f55c718c4R545 

Also, consider debouncing at least a microtask   :)